regexes/device/mobiles.yml

```
@@ -15297,10 +15307,10 @@ Nvidia:
 
 # O+ (OPlus)
 O+:
-  regex: 'O\+[ _](Air|Crunch|Ultra|Upsized|Venti|360 ALPHA PLUS 2.0)'
+  regex: 'O\+[ _](8.38z|8.97|Air|Crunch|Ultra|Upsized|Venti|360 ALPHA PLUS 2.0)'
```

Shouldn't here the dots be escaped to prevent a match for any character?
```suggestion
  regex: 'O\+[ _](8\.38z|8\.97|Air|Crunch|Ultra|Upsized|Venti|360 ALPHA PLUS 2.0)'
```